app/src/main/java/com/example/atheneum/views/adapters/NotificationListAdapter.java

```diff
@@ -0,0 +1,149 @@
+            message = (TextView) view.findViewById(R.id.notification_message);
+            time = (TextView) view.findViewById(R.id.notification_time);
+            view.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    Notification notification = getItem(getAdapterPosition());
+                    if (onClickListener != null) {
+                        onClickListener.onClick(notification);
+                    }
+                }
+            });
+        }
+    }
+
+    /**
+     * Provide interface to handle onClick events for any Notification in the list
+     */
+    public interface onClickListener {
+        public void onClick(@NonNull Notification notification);
+    }
+
+    private NotificationListAdapter.onClickListener onClickListener;
+
+    public void setOnClickListener(NotificationListAdapter.onClickListener onClickListener) {
+        this.onClickListener = onClickListener;
+    }
+
+    /**
+     * Set up ViewHolder to contain notification
+     *
+     * @param viewGroup
+     * @param position
+     * @return ViewHolder containing the notification view
+     */
+    @NonNull
+    @Override
+    public ViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int position) {
+        View view = LayoutInflater.from(viewGroup.getContext())
+                .inflate(R.layout.notification_card, viewGroup, false);
+        return new ViewHolder(view);
+    }
+
+    /**
+     * Binds Notification object to an instance of the ViewHolder
+     *
+     * @param holder ViewHolder object
+     * @param position Position of Notification within the list.
+     *                 Notification object will be bound to the ViewHolder.
+     */
+    @Override
+    public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
+        Notification notification = getItem(position);
+        holder.message.setText(notification.getMessage());
+        // Formats Date into 'how long ago'
+        // See: https://stackoverflow.com/a/3859298
+        // See: https://developer.android.com/reference/android/text/format/DateUtils.html#getRelativeTimeSpanString%28long%29
+        CharSequence momentsAgo = android.text.format.DateUtils
+                .getRelativeTimeSpanString(notification.getCreationDate().getTime());
+        holder.time.setText(momentsAgo);
+
+        if (notification.getIsSeen()) {
+            holder.notificationCard
+                    .setBackgroundColor(mContext.getResources().getColor(R.color.notificationSeen));
+        } else {
+            holder.notificationCard
+                    .setBackgroundColor(mContext.getResources().getColor(R.color.notificationNotSeen));
+        }
+    }
+
+    public void deleteItem(int position) {
+        Log.i(TAG, "SWIPED, DELETE ITEM: " + position);
+        mNotificationsViewModel.deleteNotification(position);
```

Comment:
I think it would be better if you called getItem(position) to get the notification itself, and pass the notification to the viewmodel and delete just that notification from the database directly.

Reply:
Fixed